ui/src/app/pools/components/TagHeaderForms/TagHeaderForms.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+
+import TagHeaderForms from "./TagHeaderForms";
+
+import { TagHeaderViews } from "app/tags/constants";
+
+it("can display the add tag form", () => {
+  render(
+    <TagHeaderForms
+      headerContent={{ view: TagHeaderViews.AddTag }}
+      setHeaderContent={jest.fn()}
+    />
+  );
+  expect(screen.getByText("Add tag form")).toBeInTheDocument();
```

Comment:
Could you use the `form` role instead of simple text matching please.

Reply:
This is just a placeholder until the form is implemented.

Reply:
I've added a todo here.

Reply:
thanks!